internal/config/config.go

```diff
@@ -69,6 +69,7 @@ func readInFile(v *viper.Viper, cfg *Configuration, path string) error {
 		if err != nil {
 			return errors.Wrap(ErrConfig, err.Error())
 		}
+		defer fh.Close()
```

Comment:
Not sure this has its place in a "cleanup".
At least put it in a different commit